From 57e4fb1c0b090f91e2a18a3589562369d0bb1fb8 Mon Sep 17 00:00:00 2001
From: Keerthy <j-keerthy@ti.com>
Date: Thu, 5 Sep 2019 13:50:29 +0530
Subject: [PATCH 6/8] configs: j721e_evm_r5_defconfig: Enable TPS65941 PMIC
 configs

Enable TPS65941 PMIC configs. This is needed for CPU
AVS class 0 support.

Signed-off-by: Keerthy <j-keerthy@ti.com>
Signed-off-by: Nikhil Devshatwar <nikhil.nd@ti.com>
---
 configs/j721e_evm_r5_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configs/j721e_evm_r5_defconfig b/configs/j721e_evm_r5_defconfig
index 807fea2..c2816f9 100644
--- a/configs/j721e_evm_r5_defconfig
+++ b/configs/j721e_evm_r5_defconfig
@@ -105,10 +105,12 @@ CONFIG_TI_SCI_POWER_DOMAIN=y
 CONFIG_DM_PMIC=y
 CONFIG_SPL_PMIC_PALMAS=y
 CONFIG_SPL_PMIC_LP87565=y
+CONFIG_PMIC_TPS65941=y
 CONFIG_DM_REGULATOR=y
 CONFIG_SPL_DM_REGULATOR=y
 CONFIG_SPL_DM_REGULATOR_PALMAS=y
 CONFIG_SPL_DM_REGULATOR_LP87565=y
+CONFIG_DM_REGULATOR_TPS65941=y
 CONFIG_K3_AVS0=y
 CONFIG_K3_SYSTEM_CONTROLLER=y
 CONFIG_REMOTEPROC_TI_K3_ARM64=y
-- 
2.7.4

